src/components/NextQuestion/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+
+import { SurveyAnswerContext } from 'contexts/surveyAnswer';
+import nextIcon from 'images/next-button-black.svg';
+
+const NextQuestion = ({ questionId, nextQuestion }) => {
+  const answerContext = useContext(SurveyAnswerContext)
+  const answerObject = answerContext.state.questions.find(obj => obj.id === questionId);
+
+  // Checking if the answer is selected for this question
+  // if not does not go to next question
+  const answerSelected = () => {
+    if (answerObject && answerObject.answers.length !== 0) {
```

Comment:
```suggestion
    if (answerObject && answerObject.answers.length > 0) {
```

IMO, this will be more understanding when reading the code. When the length can be only one direction (which is only positive in this case), it would be better to compare them with the appropriate operator.